public/locales/en/translation.json

```diff
@@ -147,6 +147,7 @@
     "revertPrice": "Revert price",
     "scanningPeers": "Scanning peers on the network",
     "searchByNameOrAddress": "Search name or paste address",
+    "selectedServer": "Querying the selected server...",
```

Comment:
```suggestion
    "selectedServer": "Selected {{serverUrl}} as custom server",
```

`Querying the selected server...` suggests it's in a loading state. But design will take a look at this to communicate this better.

Reply:
I committed this change, but `{{serverUrl}}` isn't pulling in any dynamic variables

Reply:
I fixed this up, but the issue is that the dynamic variable comes out as a URL. I'm not sure if servers have names that are nice to read. Originally @dmosites mentioned to say `"Querying the selected server..."`. How should we proceed with this?

<img width="508" alt="Screenshot 2023-06-01 at 4 45 49 PM" src="https://github.com/airswap/airswap-web/assets/90465420/89d4dc23-493d-425d-a138-7a8ee3473caa">